packages/stark-ui/src/modules/dropdown/components/dropdown.component.html

```diff
@@ -1,63 +1,60 @@
-<mat-form-field>
-	<!-- single-selection view -->
-	<mat-select
-		*ngIf="!isMultiSelectEnabled"
-		[id]="dropdownId"
-		[formControl]="formControl"
-		[placeholder]="placeholder | translate"
-		[attr.aria-label]="dropdownName | translate"
-		[required]="required"
-	>
-		<!-- FIXME find a replacement for md-select-header as mat-select-header does not exist yet https://github.com/angular/material2/pull/7835 -->
-		<!--<mat-select-header-->
-		<!--*ngIf="header !== undefined && header !== ''"-->
-		<!--class="stark-dropdown-header"-->
-		<!--translate>-->
-		<!--{{header}}-->
-		<!--</mat-select-header>-->
+<!-- single-selection view -->
+<mat-select
+	*ngIf="!multiSelect"
+	[value]="value"
+	[id]="dropdownId"
+	[placeholder]="placeholder | translate"
+	[attr.aria-label]="dropdownName | translate"
+	[required]="required"
+	[disabled]="disabled"
+	(selectionChange)="onSelectionChange($event)"
+	(openedChange)="onOpenedChange()"
+>
+	<!-- FIXME find a replacement for md-select-header as mat-select-header does not exist yet https://github.com/angular/material2/pull/7835 -->
+	<!--<mat-select-header-->
+	<!--*ngIf="header !== undefined && header !== ''"-->
+	<!--class="stark-dropdown-header"-->
+	<!--translate>-->
+	<!--{{header}}-->
+	<!--</mat-select-header>-->
 
-		<mat-option *ngIf="defaultBlank" [value]=""></mat-option>
+	<mat-option *ngIf="defaultBlank" [value]=""></mat-option>
 
-		<span>
-			<mat-option *ngFor="let option of options; trackBy: trackItemFn" [value]="getOptionValue(option)">
-				{{ getOptionLabel(option) | translate }}
-			</mat-option>
-		</span>
-	</mat-select>
+	<span>
+		<mat-option *ngFor="let option of options; trackBy: trackItemFn" [value]="getOptionValue(option)">
+			{{ getOptionLabel(option) | translate }}
+		</mat-option>
+	</span>
+</mat-select>
 
-	<!-- multiple-selection view -->
-	<mat-select
-		*ngIf="isMultiSelectEnabled"
-		[id]="dropdownId"
-		[formControl]="formControl"
-		[placeholder]="placeholder | translate"
-		[attr.aria-label]="dropdownName | translate"
-		[required]="required"
-		multiple
-	>
-		<!-- FIXME find a replacement for md-select-header as mat-select-header does not exist yet https://github.com/angular/material2/pull/7835 -->
-		<!--<mat-select-header-->
-		<!--ng-if="$ctrl.header !== undefined && $ctrl.header !== ''"-->
-		<!--class="stark-dropdown-header"-->
-		<!--translate>-->
-		<!--{{header}}-->
-		<!--</mat-select-header>-->
+<!-- multiple-selection view -->
+<mat-select
+	*ngIf="multiSelect"
+	[value]="value"
+	[id]="dropdownId"
+	[placeholder]="placeholder | translate"
+	[attr.aria-label]="dropdownName | translate"
+	[required]="required"
+	[disabled]="disabled"
+	(selectionChange)="onSelectionChange($event)"
+	(openedChange)="onOpenedChange()"
+	multiple
```

Comment:
I think if you use `[multiple]="multiselect"` there is no need to repeat the code.

Reply:
Yes and no. Thanks to our implementation, we are able to switch from `multi` to not multi without any errors, which is not the case of angular `mat-select` 😊

Reply:
Ah, yes indeed. Could you add a comment somewhere explaining (linking) this behaviour? Maybe [this](https://github.com/angular/material2/blob/master/src/lib/select/select.md#error-cannot-change-multiple-mode-of-select-after-initialization) page?

Reply:
Indeed, but I would link to the official Docs website instead: https://material.angular.io/components/select/overview#error-cannot-change-code-multiple-code-mode-of-select-after-initialization